redbot/cogs/audio/core/commands/player.py

```diff
@@ -37,7 +37,11 @@ class PlayerCommands(MixinMeta, metaclass=CompositeMetaClass):
     @commands.guild_only()
     @commands.bot_has_permissions(embed_links=True)
     async def command_play(self, ctx: commands.Context, *, query: str):
-        """Play a URL or search for a track."""
+        """Play a URL or search for a track.
+
+        To play a local track/file, the query should be <parentfolder>/<filename>.
+        If you are the bot owner, use [p]audioset info to display your localtracks path.
```

Comment:
given that 99.99% of the users using this wont be the bot owner; please remove this.